web/app/globals.css

```
@@ -1,70 +1,120 @@
+  min-height: 100%;
+  color: var(--foreground);
+  background:
+    radial-gradient(circle at 8% 2%, rgba(56, 189, 248, 0.15), transparent 32%),
+    radial-gradient(circle at 100% 100%, rgba(34, 197, 94, 0.16), transparent 28%),
+    linear-gradient(180deg, #f8fbff, #eef3fb 42%, #eef2ff 100%);
+  font-family: var(--font-body);
   letter-spacing: -0.01em;
   -webkit-font-smoothing: antialiased;
-  height: 100%;
-  overflow-y: auto;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.7.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)